From c206ccb2691ebbbe311345e0d2e84bbe0674fc39 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 19 Feb 2020 14:39:11 +0200
Subject: [PATCH] [SW-9764] gaudi: pll: lower pci div out1 by half

There has been a h/w bug discovered in the trace/profiler integration.
To W/A the bug, we need to lower the PCI PLL DIV OUT1 to half of the
actual PCI PLL (DIV OUT0) frequency.
Lower the PCI PLL DIV OUT1 frequency by half.

Change-Id: Icbf4a8db357ffe990ca15e113050bbbdc7467391
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/asic.c b/gaudi/asic.c
index ffbc087..e6e23e2 100644
--- a/gaudi/asic.c
+++ b/gaudi/asic.c
@@ -102,9 +102,9 @@
 		.name = "PCI", .base = mmPSOC_PCI_PLL_BASE,
 //		.nr = 0, .nf = 37, .od = 1, .nb = 18, /* 950 */
 		.nr = 4, .nf = 331, .od = 3, .nb = 165, //DAY 1 (830MHz)
-		.div_factor = { 0, 0, 4, 9 },
+		.div_factor = { 0, 1, 4, 9 },
 		.div_sel = { HL_PLL_PLL_CLK,
-			     HL_PLL_PLL_CLK,
+			     HL_PLL_DIV_PLL,
 			     HL_PLL_DIV_PLL,
 			     HL_PLL_DIV_PLL
 		},
